build-repo.sh

```diff
@@ -360,6 +360,23 @@ for manifest in ${manifests}; do
   fi
 done
 
+# Extract dependency error logs
+dep_errors=$(sed -n '/^ERROR: unable to select packages/{p; :loop; n; /^\s/{p; b loop}; /^>>>/{b loop}}' ${full_log_file})
+dep_error_lines=$(echo -n "${dep_errors}" | wc -l)
+echo "${dep_error_lines}"
+if [ ${dep_error_lines} -gt 0 ]; then
+  echo >> ${summary_file}
+  echo '---' >> ${summary_file}
+  echo >> ${summary_file}
+  echo "## Dependency logs" >> ${summary_file}
+  if [ ${dep_error_lines} -gt 50 ]; then
```

Comment:
It's a detail but the maximum number of lines is 50 but we echo up to 100 lines to the summary file.